### PR TITLE
Fleshed out the narrative for why security code analysis matters

### PR DESCRIPTION
**Pull request by andyblundell**

### CODE REVIEW

File: quality-checks.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ We recommend tracking progress on an Engineering Quality dashboard, for example:
...
+| Security code analysis | Security        | Universal     | Check for indications of possible security issues (for example injection weaknesses)                                                                                                                            | This gives fast feedback about security issues: it's not as thorough as security testing in terms of finding complex weaknesses, but has much greater coverage than security testing (so it's a better option for finding simple weaknesses) and it's much quicker to execute, so both are important to achieve both rapid and thorough security testing| If using SonarQube, must use SonarQube's default [rules, profiles and gateways](tools/sonarqube.md#default-quality-gates) <br/><br/> Build pipeline must fail if gateway is not met | One option is [SonarQube](tools/sonarqube.md). For the purpose of security code analysis, Developer Edition or higher is required as it includes advanced OWASP scanning. |                                                                                                               |
```
Comment (walteck) [inline quality-checks.md:48]: suggest rewording: "This gives fast feedback about security issues: it's not as thorough as security testing in terms of finding complex weaknesses, but has much greater coverage than security testing (so it's a better option for finding simple weaknesses) and it's much quicker to execute, so both are important to achieve both rapid and thorough security testing" 
to 
"This gives fast feedback about security issues. Code analysis is not as thorough as security testing in terms of finding complex weaknesses, but has much greater coverage. It's a better option for finding simple weaknesses and it's much quicker to execute. Security code analysis and security testing are both important to achieve rapid and thorough security testing."

**Review by walteck — COMMENTED** [review]
Suggested some rewording.

File: quality-checks.md (hunk 1 above)
Comment (andyblundell) [reply quality-checks.md:48]: That's much better wording, thanks @walteck

Comment (andyblundell) [reply quality-checks.md]: I've added the note about runtime only - any more for any more?

Comment (walteck) [reply quality-checks.md]: LGTM
